WcaOnRails/app/models/user.rb

```diff
@@ -1259,4 +1259,48 @@ def is_delegate_in_probation
   def region
     UserGroup.find_by_id(self.region_id)
   end
+
+  def can_manage_delegate_probation?
```

Comment:
Oh hey, this seems like it's exactly the kind of method that I wanted you to use in my other lengthy comment above.

Reply:
Thanks!